### PR TITLE
Personal Artificial Intelligence Fix

### LINKED ISSUE #13274: There is no prompt that pAI personalities are available on actively searching pAI devices
I like downloading pAIs relatively often, and on occasion playing them, but it always feels like guesswork when one is ready to be downloaded.

Back when I played on /vg/ I'd get a prompt when I could download one, but I've never heard or seen the prompt on tg when I try for pAIs.

I rooted around the code and found that the code for the prompt exists in both, but I've never gotten it on tg before. Am I deaf / blind or is the proc simply not applying on tg?

### PR DESCRIPTION
**Pull request by Jalleo**
Fixes #13274

Not sure if it tells everyone around or not as I cant test it on my own.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -9,4 +9,5 @@
 *.lk
 *.int
 *.backup
-*.int
\ No newline at end of file
+*.int
+/*.dme
```
Comment (Jalleo) [inline .gitignore]: I fudged up and need to remove this gonna work out what I did and undo it then sort that

**Comment by GunHog** [comment]
How to test:
Compile code.
Load Dream Daemon.
Join server.
Logout of BYOND and join again, resulting in dual login.
Start the round on your account, playing as a ghost
Join as observer on logged in account, making sure you pAI flag is enabled.
Get a pAI and activate it, then submit the application entry on your ghost login.

**Comment by GunHog** [comment]
Let me try that again, writing properly:
Join as human on the guest login
Join as ghost on the account.

**Comment by Jalleo** [comment]
Okay well I shall do that tomorrow then

**Comment by Lo6a4evskiy** [comment]
>How to test:
Compile code.
Load Dream Daemon.
Join server.
Logout of BYOND and join again, resulting in dual login.
Start the round on your account, playing as a ghost
Join as human on logged in account, making sure your pAI flag is enabled.
Get a pAI and activate it, then submit the application entry on your ghost login.

Oh god that is amazing thank you so much

File: .gitignore [hunk 2]
```diff
@@ -9,4 +9,5 @@
 *.lk
 *.int
 *.backup
-*.int
\ No newline at end of file
+*.int
+
```
Comment (tkdrg) [inline .gitignore]: ????

Comment (Jalleo) [inline .gitignore]: ill fix it tomorrow it was a change to this file that I didnt mean to have commit I removed it but git didnt think I removed the line...

**Comment by duncathan** [comment]
>7 commits
>one intended line change

wew

**Comment by Jalleo** [comment]
Most of them was reverting local changes to my master to make it clean again.

File: code/game/objects/items/devices/paicard.dm [hunk 3]
```diff
@@ -128,7 +128,7 @@
 			if(10) src.overlays += "pai-null"
 
 /obj/item/device/paicard/proc/alertUpdate()
-	visible_message("<span class ='info'>[src] flashes a message across its screen, \"Additional personalities available for download.\"", 3, "<span class='notice'>[src] bleeps electronically.</span>", 2)
+	audible_message("<span class ='info'>[src] flashes a message across its screen, \"Additional personalities available for download.\"", 3, "<span class='notice'>[src] bleeps electronically.</span>", 2)
```
Comment (Jalleo) [inline code/game/objects/items/devices/paicard.dm]: @phil235 the bug is the message is not actually working anymore the proc activates but the message no longer works something broke this elsewhere.

**Comment by Jalleo** [comment]
Right this is now working again the visible message was working out of new parameters we need a overflow parameter added to report of any other bugged ones so we can issue report. Also visible message doesn't work correctly if the object is on the person.

File: code/game/objects/items/devices/paicard.dm [hunk 4]
```diff
@@ -128,7 +128,8 @@
 			if(10) src.overlays += "pai-null"
 
 /obj/item/device/paicard/proc/alertUpdate()
-	visible_message("<span class ='info'>[src] flashes a message across its screen, \"Additional personalities available for download.\"", 3, "<span class='notice'>[src] bleeps electronically.</span>", 2)
+	world << "damn you its broken"
```
Comment (Jalleo) [inline code/game/objects/items/devices/paicard.dm]: Whoops debug sorry

**Comment by duncathan** [comment]
>more commits than characters changed

**Pull request merged**